MachOKit/Core/Memory/MKMemoryMap.m

```diff
@@ -130,15 +128,15 @@ - (vm_size_t)copyBytesAtOffset:(mk_vm_offset_t)offset fromAddress:(mk_vm_address
     __block NSError *localError;
     
     [self remapBytesAtOffset:offset fromAddress:contextAddress length:length requireFull:requireFull withHandler:^(vm_address_t address, vm_size_t mappingLength, NSError *error) {
-        localError = error;
+        localError = [error retain];
         if (error)
             return;
         
         memcpy(buffer, (void*)address, (vm_size_t)MIN(length, (mk_vm_size_t)mappingLength));
         retValue = (vm_size_t)length;
     }];
     
-    MK_ERROR_OUT = localError;
+    MK_ERROR_OUT = [localError autorelease];
```

Comment:
Good find.  This change should be applied to all of the convenience methods in MKMemoryMap.m that pass the error from `-remapBytesAtOffset:fromAddress:length:requireFull:withHandler:` back to the caller.

Reply:
Fixed in a106c71